unit/FFT/FFT.cpp

```diff
@@ -33,7 +33,9 @@ int main() {
           data(k,j,i) = i+1;
         });
 
+        yakl::timer_start("fft.forward_real");
```

Comment:
@omarkahmed , can you wrap these in `#ifdef YAKL_AUTO_PROFILE` ? I only want these timed if that is specified. Thanks.